scikit/notebooks/Scikit Learn on CMLE.ipynb

```diff
@@ -0,0 +1,702 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Online Prediction with Scikit-Learn on Google Cloud Machine Learning Engine\n",
+    "Alright, so you’ve built your machine learning models with scikit-learn, you want to serve your models in real time for a real application, but managing infrastructure sounds like a nightmare. Thankfully, Google now has the answer. Google supports serving your trained models on Cloud Machine Learning Engine (ML Engine) with online prediction.\n",
```

Comment:
What does "real application" mean?

Reply:
I don't have a good answer, so I'm gonna cut it.